### DISCUSSION TITLE
请问waline依然必须依赖leancloud吗？

### DISCUSSION THREAD

**Discussion by a1401358759** [walinejs/waline#692] [Q&A] [answered]
请问waline依然必须依赖leancloud吗？使用腾讯云开发部署之后，环境变量里依然要填写leancloud注册的key和secret等？

**Comment by lizheming** [comment, marked as answer]
不需要的，云开发部署的话可以参考这篇文档 https://waline.js.org/guide/server/cloudbase.html

**Reply by a1401358759** [reply]
感谢 因为我就是看到这篇文章最后的截图里面环境变量配置了LEAN_ID和LEAN_KEY，以为必须依赖leancloud呢

**Reply by lizheming** [reply]
嗯，我把配图改改吧。云开发默认不用配置任何环境变量就可以启动了。

**Reply by frank1998sj** [reply]
[这个文档](https://waline.js.org/reference/server.html#%E4%B8%BB%E8%A6%81%E9%85%8D%E7%BD%AE)里仍旧把 LEAN_ID、LEAN_KEY 和 LEAN_MASTER_KEY 标注为必填的环境变量

**Reply by lizheming** [reply]
@frank1998sj 这里是针对快速开始用户提供的选项，如果不把它设置成必填，可能会造成不需要任何数据库相关的配置的错觉。

**Comment by aiokr** [comment]
如果使用 vercel 部署，链接 cloudbase 数据库，还需要依赖 leancloud 吗

**Reply by lizheming** [reply]
不用，不过建议直接都整体使用 cloudbase 比较好
